mathjax3-ts/input/tex/cancel/CancelConfiguration.ts

```diff
@@ -0,0 +1,98 @@
+/*************************************************************
+ *
+ *  Copyright (c) 2018 The MathJax Consortium
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+
+/**
+ * @fileoverview Configuration file for the cancel package.
+ *
+ * @author v.sorge@mathjax.org (Volker Sorge)
+ */
+
+import {Configuration} from '../Configuration.js';
+import TexParser from '../TexParser.js';
+import {TexConstant} from '../TexConstants.js';
+import {CommandMap} from '../SymbolMap.js';
+import {ParseMethod} from '../Types.js';
+import ParseUtil from '../ParseUtil.js';
+
+
+/**
+ * The attributes allowed in \enclose{notation}[attributes]{math}
```

Comment:
Is this supposed to refer to `\canel` rather than `\enclose`?

Reply:
Actually no, as it accepts the same parameters as the `\enclose` macro. But the order is wrong.